### ISSUE TITLE
implprefix DeprecationWarning with pluggy 0.7.1

### ISSUE THREAD

**Issue by edmorley** [mozilla/PyPOM#90]
Hi!

[pluggy](https://pypi.org/project/pluggy) v0.7.1 has deprecated the `implprefix` argument:
* https://github.com/pytest-dev/pluggy/blob/master/CHANGELOG.rst#pluggy-071-2018-07-28
* https://github.com/pytest-dev/pluggy/pull/144

Treeherder marks warnings as errors, which causes the following failures from PyPOM:

```
DeprecationWarning: Support for the `implprefix` arg is now deprecated and
will be removed in an upcoming release. Please use HookimplMarker.
```
(Full log: https://travis-ci.org/mozilla/treeherder/jobs/409417404#L901)

The usage within PyPOM is here:
https://github.com/mozilla/PyPOM/blob/697ae6a0115fc05af071d4e41a38da71fae33189/pypom/view.py#L20

We can temporarily ignore this `DeprecationWarning` via pytest's `filterwarnings`, however it would be good to fix upstream at some point.

**Comment by davehunt** [comment]
@edmorley this should be fixed by #92 but I'm on PTO for a week from tomorrow so I'd rather not do a release when I'm not around to deal with any potential issues. Are you okay with waiting on this?

**Comment by edmorley** [comment]
Yeah no rush at all (we've added a temporary `filterwarnings` entry for it); thanks for opening the PR :-)

**Comment by edmorley** [comment]
Many thanks :-)
